### ISSUE TITLE
Several Games pages are inaccessible

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#1384]
- Hangman - does not work with screen readers
- Memory game - does not work with screen readers
- Wordsearch - does not work with screen readers or keyboard only

These issues will be noted in the updated accessibility statement but logging here as ideally they would be more accessible (not sure how wordsearch could be made screen reader accessible in a way thatis nice to use though...)

**Comment by FayCross** [comment]
For info, these issues are noted in the 3.14 accessibility statement:

In non-compliance with the accessibility regulations section:
Wordsearches in XOT projects can only be completed by selecting letters using a mouse click and drag motion. These pages are inaccessible to sighted, keyboard only users and those who cannot perform dragging movements precisely. This fails [2.1.1 (Keyboard)](https://www.w3.org/WAI/WCAG22/quickref/?showtechniques=222%2C211#keyboard-accessible) and [2.5.7 (Dragging Movements)](https://www.w3.org/WAI/WCAG22/quickref/?showtechniques=144%2C1410%2C211%2C222%2C247#dragging-movements).

In non accessible and disproportionate burden section:
Some interactivities in XOT projects are visually focused and add value for sighted users whilst being very challenging for screen reader users. These include the following pages: Image Sequence, Sortable Grid, Hangman, Memory Game and Wordsearch. Discussions with the Xerte Project team and accessibility specialists suggest that the value obtained by screen reader users in doing these activities is minimal in relation to the opportunity cost of diverting volunteer effort from other developments that would add more value to more users. 

Alternative versions of these activities should be provided by project authors, as required, to enable an “equivalence of experience” between screen reader and sighted users.
